js/everything.js

```diff
@@ -218,6 +227,8 @@ var ZenPlayer = {
                     }
                     window.sessionStorage[videoID] = resumeTime;
                 }
+                var updatedUrl = that.videoUrl + "&t=" + parseInt(resumeTime, 10);
```

Comment:
Actually, `if (rescueTime <= 0)` we should do `$("#zen-video-title").attr("href", that.videoUrl);`. To be effecient though, we should only do it `if ($("#zen-video-title").attr("href") !== that.videoUrl)`.